Add installation instructions in cmake files.
Install hypseus binary and resources (sounds, pictures and font) in /usr/share/hypseus

Signed-off-by: Laurent Merckx <laurent-merckx@skynet.be>

--- a/src/CMakeLists.txt	2018-04-26 21:46:16.815307867 +0200
+++ b/src/CMakeLists.txt	2018-04-26 21:45:56.547520309 +0200
@@ -129,3 +129,7 @@ set(CPACK_PACKAGE_VERSION_MINOR "${VER_M
 set(CPACK_PACKAGE_VERSION_PATCH "${VER_PATCH}")
 
 include(CPack)
+
+execute_process(COMMAND svn export --username svn --password svn https://www.daphne-emu.com:9443/daphnesvn/branches/v_1_0/sound ../sound)
+install(TARGETS hypseus DESTINATION bin)
+install(DIRECTORY ../pics ../fonts ../sound DESTINATION share/daphne)
